cl/corpus_importer/management/commands/columbia_merge.py

```diff
@@ -370,8 +370,12 @@ def merge_docket_data(docket_data: dict, cluster: OpinionCluster) -> None:
     data_to_update = {}
 
     if docket_data["docket_number"]:
-        merge_docket_numbers(cluster, docket_data["docket_number"])
-        cluster.docket.refresh_from_db()
+        new_docket_number = merge_docket_numbers(
```

Comment:
can you provide an example for this?  why is this being added

Reply:
There are clusters without docket number, in some of the matches from the full list, for example this cluster: https://www.courtlistener.com/opinion/1439900/go/ and this file and docket number  "E2000-02272-COA-R3-CV" from file matched tennessee\court_opinions\documents\4c52f83b11e551bb.xml

The function could be reduced to that if there is no docket number in the system then save the one from the file.